brian2/importexport/importexport.py

```diff
@@ -0,0 +1,74 @@
+import abc
+from abc import abstractmethod, abstractproperty
+
+class ImportExport(object):
+    __metaclass__ = abc.ABCMeta
+    
+    #: A dictionary mapping import/export methods names to `ImportExport` objects
+    methods = dict()
+
+    @staticmethod    
+    def register(name, importerexporter):
+        '''
+        Register a import/export method. Registered importerexporter can be referred to
+        via their name.
+        
+        Parameters
+        ----------
+        name : str
+            A short name for the import/export class (e.g. `'dict'`)
+        importerexporter : `ImportExport`
+            The importerexporter object, e.g. an `DictImportExport`.
+        '''
+        
+        name = name.lower()
+        if name in ImportExport.methods:
+            raise ValueError(("An import/export methods with the name {}" +\
+                             "has already been registered").format(name))
+
+        if not isinstance(importerexporter, ImportExport):
+            raise ValueError(('Given importerexporter of type {} does not seem to ' +\
```

Comment:
same here for the line continuation. Also for consistency (we are not very consistent over the Brian 2 code base, though...) use either single or double quotes for strings in the function.